sound/soc/sof/intel/skl.c

```diff
@@ -85,6 +85,10 @@ struct snd_sof_dsp_ops sof_skl_ops = {
 	/* firmware loading */
 	.load_firmware = hda_dsp_cl_load_fw,
 
+	/* pre/post fw load */
+	.pre_fw_run = hda_dsp_pre_fw_run,
```

Comment:
run?

Reply:
@plbossart I stuck to run because we do this in snd_sof_run_firmware().

does pre_load() and post_boot() make more sense?

Reply:
no I meant that your comments use fw run everywhere except where I flagged you are still referring to fw load

Reply:
fixed!